static/js/search_suggestion.js

```diff
@@ -40,6 +40,37 @@ function match_criteria(operators, criteria) {
     });
 }
 
+function compare_by_huddle(huddle) {
+    huddle = _.map(huddle.slice(0, -1), function (person) {
+        person = people.get_by_email(person);
+        if (person) {
+            return person.user_id;
+        }
+    });
+
+    // Construct dict for all huddles, so we can lookup each's recency
+    var huddles = activity.get_huddles().slice(0, 10);
```

Comment:
I think we're unlikely to have a lot of old huddles in the frontend in the first place, but I think even a 1-month-old huddle is much more likely to be who you're trying to talk to than a random user :).